task/build-paketo-ubi-builder/0.1/build-paketo-ubi-builder.yaml

```diff
@@ -0,0 +1,527 @@
+        workingDir: /var/workdir
+      - computeResources:
+          limits:
+            cpu: "4"
+            memory: 4Gi
+          requests:
+            cpu: "1"
+            memory: 1Gi
+        image: quay.io/konflux-ci/buildah-task:latest@sha256:b2d6c32d1e05e91920cd4475b2761d58bb7ee11ad5dff3ecb59831c7572b4d0c
+        name: inject-sbom-and-push
+        script: |
+          #!/bin/bash
+          set -e
+          if [ "${IMAGE_APPEND_PLATFORM}" == "true" ]; then
+            IMAGE="${IMAGE}-${PLATFORM//[^a-zA-Z0-9]/-}"
+            export IMAGE
+          fi
+          
+          retry() {
+            status=-1
+            max_run=5
+            sleep_sec=10
+          
+            for run in $(seq 1 $max_run); do
+              status=0
+              [ "$run" -gt 1 ] && sleep $sleep_sec
+              "$@" && break || status=$?
+            done
+            return $status
+          }
+          
+          ca_bundle=/mnt/trusted-ca/ca-bundle.crt
+          if [ -f "$ca_bundle" ]; then
+            echo "INFO: Using mounted CA bundle: $ca_bundle"
+            cp -vf $ca_bundle /etc/pki/ca-trust/source/anchors
+            update-ca-trust
+          fi
+          
+          echo "##########################################################################################"
+          echo "Pull the image from the OCI storage."
+          echo "##########################################################################################"
+          buildah --storage-driver $STORAGE_DRIVER pull $IMAGE
+          
+          echo "##########################################################################################"
+          echo "## Copy within the container of the image the sbom files"
+          echo "##########################################################################################"
+          #base_image_name=$(buildah inspect --format '{{ index .ImageAnnotations "org.opencontainers.image.base.name"}}' $IMAGE | cut -f1 -d'@')
+          #base_image_digest=$(buildah inspect --format '{{ index .ImageAnnotations "org.opencontainers.image.base.digest"}}' $IMAGE)
+          
+          container=$(buildah --storage-driver $STORAGE_DRIVER from --pull-never $IMAGE)
+          buildah --storage-driver $STORAGE_DRIVER copy $container sbom-cyclonedx.json sbom-purl.json /root/buildinfo/content_manifests/
+          
+          # The image build don't include "ImageAnnotations" as "null"
+          #buildah config -a org.opencontainers.image.base.name=${base_image_name} -a org.opencontainers.image.base.digest=${base_image_digest} $container
+          
+          BUILDAH_ARGS=()
+          if [ "${SQUASH}" == "true" ]; then
+            BUILDAH_ARGS+=("--squash")
+          fi
+          
+          buildah --storage-driver $STORAGE_DRIVER commit "${BUILDAH_ARGS[@]}" $container $IMAGE
+          set +x
+          echo "######################################################$TLSVERIFY####################################"
```

Comment:
Why is TLSVERIFY just randomly printed here?

Reply:
Typo error. Removed: 5457c773